development/compiling/compiling_for_macos.rst

```diff
@@ -100,38 +100,38 @@ Compiling a headless/server build
 To compile a *headless* build which provides editor functionality to export
 projects in an automated manner, use::
 
-    scons platform=server tools=yes target=release_debug --jobs=$(sysctl -n hw.logicalcpu)
+    scons platform=server target=editor --jobs=$(sysctl -n hw.logicalcpu)
```

Comment:
`server` platform no longer exist, it's replaced with `--headless` command line argument of the normal executable.

Reply:
I tried to update the section with minimal changes, please review the wording